src/core/services/nexus/stream/posts/postStream.api.ts

```diff
@@ -58,6 +58,10 @@ export const postStreamApi = {
   replies: (params: Core.TStreamPostRepliesParams) =>
     buildPostStreamUrl(params, Core.StreamSource.REPLIES, Core.STREAM_PREFIX.POSTS),
 
+  // Alias for replies (for backwards compatibility)
+  postReplies: (params: Core.TStreamPostRepliesParams) =>
+    buildPostStreamUrl(params, Core.StreamSource.REPLIES, Core.STREAM_PREFIX.POSTS),
+
```

Comment:
@tipogi is this key the right one?

`postReplies` or `post_replies`

Reply:
https://github.com/pubky/franky/pull/408/commits/5131dbb9d365c9f83b0acde8adf60795f60faaf7